packages/cart/src/models/line-item.ts

```diff
@@ -144,10 +148,28 @@ export default class LineItem {
   @BeforeCreate()
   onCreate() {
     this.id = generateEntityId(this.id, "cali")
+
+    const val = new BigNumber(this.raw_unit_price ?? this.unit_price)
+
+    this.unit_price = val.numeric
+    this.raw_unit_price = val.raw!
```

Comment:
I believe we can create a util to do that for 1 or N fields.

`modelToBigNumber(this, "unit_price")`
or
`modelToBigNumber(this, ["unit_price", "quantity"])` 

subliminal suggestion to set quantity as BigNumber 😆

Reply:
That would make the `raw_` prefix a strict requirement right? Otherwise, I am not sure how we can safely do `this.raw_... ?? this....`